### DISCUSSION TITLE
What company HQ is located at this address?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#728] [Galaxy Trivia] [answered]
1 Hacker Way, Menlo Park, CA

**Comment by SANKARAMDAS** [comment, marked as answer]
Meta Headquarters
Initially 1 Hacker Way. Apple has 1 Infinite Loop, Genentech has 1 DNA Way, and now Facebook has its own vanity headquarters address: 1 Hacker Way, Menlo Park sit on the edge of San Francisco Bay, 94025.

**Comment by peterbaccaro** [comment]
1 Hacker Way, Menlo Park, CA is the headquarters campus of **Meta** and its **Facebook** subsidiary.
It was also the former headquarters of Sun Microsystems.

https://www.wikidata.org/wiki/Q58814026
